be1-go/hub/election.go

```diff
@@ -23,6 +23,9 @@ type electionChannel struct {
 	// Questions asked to the participants
 	//the key will be the string representation of the id of type byte[]
 	questions map[string]question
+
+	// attendees that took part in the roll call string of their PK
+	attendees []string
```

Comment:
I think you can use a map here as it is done in the LaoChannel: `attendees map[string]struct{}`. It will make the check easier.

Reply:
done